games-action/supertuxkart/files/supertuxkart-0.9.2-angelscript-compatibility.patch

```diff
@@ -0,0 +1,325 @@
+From 6289b5fc298ac4bb48f426e3325a9bf514a603f1 Mon Sep 17 00:00:00 2001
+From: Igor Gnatenko <ignatenko@redhat.com>
+Date: Tue, 21 Jun 2016 11:48:13 +0200
+Subject: [PATCH] make code compatible with angelscript 2.31.0
```

Comment:
supertuxkart is a rather friendly upstream, has this patch been communicated with upstream?